packages/backend/src/index.ts

```diff
@@ -179,8 +177,8 @@ app.use((error: Error, req: Request, res: Response, _: NextFunction) => {
 // Run the server
 const port = process.env.PORT || 8080;
 app.listen(port, () => {
-    console.log(
-        `   |     |     |     |     |     |     |\n   |     |     |     |     |     |     |\n   |     |     |     |     |     |     |  \n \\ | / \\ | / \\ | / \\ | / \\ | / \\ | / \\ | /\n  \\|/   \\|/   \\|/   \\|/   \\|/   \\|/   \\|/\n------------------------------------------\nLaunch lightdash at http://localhost:${port}\n------------------------------------------\n  /|\\   /|\\   /|\\   /|\\   /|\\   /|\\   /|\\\n / | \\ / | \\ / | \\ / | \\ / | \\ / | \\ / | \\\n   |     |     |     |     |     |     |\n   |     |     |     |     |     |     |\n   |     |     |     |     |     |     |`,
+    Logger.debug(
```

Comment:
wouldn't this be an `info` message?

Reply:
I believe this won't show if we set to log level `warning` (which happens in `logger.ts`)

Log levels: 
```
{ 
  emerg: 0, 
  alert: 1, 
  crit: 2, 
  error: 3, 
  warning: 4, 
  notice: 5, 
  info: 6, 
  debug: 7
}
```